packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/semanticdata/services/api/ISemanticDataCreationService.java

```diff
@@ -16,11 +16,15 @@
 import org.eclipse.sirius.web.domain.boundedcontexts.project.Project;
 import org.springframework.data.jdbc.core.mapping.AggregateReference;
 
+import java.util.UUID;
+
 /**
  * Used to create the semantic data.
  *
  * @author sbegaudeau
  */
 public interface ISemanticDataCreationService {
     void initialize(ICause cause, AggregateReference<Project, String> project);
+
+    void add(ICause cause, UUID semanticDataId, AggregateReference<Project, String> project);
```

Comment:
Can be removed but we will probably need to update the previous method to something like create which would return the created semantic data